extensions/wikia/AdEngine/js/provider/directGptMobile.js

```diff
@@ -15,13 +15,23 @@ define('ext.wikia.adEngine.provider.directGptMobile', [
 			INVISIBLE_HIGH_IMPACT:      {size: '1x1'},
 			INVISIBLE_HIGH_IMPACT_2:    {loc: 'hivi'},
 			MOBILE_TOP_LEADERBOARD:     {size: '300x50,320x50,320x100,320x480,2x2', loc: 'top'},
-			BOTTOM_LEADERBOARD:         {size: '320x50,300x250,300x50,2x2', loc: 'footer', pos: ['BOTTOM_LEADERBOARD', 'MOBILE_PREFOOTER']},
+			BOTTOM_LEADERBOARD:         {
+				size: '320x50,300x250,300x50,2x2',
+				sizeMap: adContext.get('opts.additionalBLBSizes') ? [
+					{
+						viewport: [375, 627],
+						sizes: [[2, 2], [300, 50], [320, 50], [300, 250], [300, 600]]
```

Comment:
Is it safe for UAP/non-UAP scenarios? I wonder if adSizeFilter isn't used too early here:
https://github.com/Wikia/app/pull/15112/files#diff-1adab53966667a993505d6e769ad9c60R20

Reply:
if normal sizes are filtered 3 lines below, why it would be too early?

Reply:
kk, didn't catch that